assets/src/components/checkboxInput/index.js

```diff
@@ -0,0 +1,16 @@
+import { CheckboxControl, Tooltip } from '@wordpress/components';
+
+/**
+ * Muriel-styled Checkbox.
+ */
+class CheckboxInput extends CheckboxControl {
```

Comment:
How should we name the components? They can't have the same name as the `@wordpress/components` component, but it may be confusing if they all have a slightly different name (e.g. `CheckboxInput` and `CheckboxControl`). Should we just prefix the `@wordpress/components` class (e.g. `NewspackCheckboxControl`)?